src/App.js

```diff
@@ -1,10 +1,47 @@
 import React from 'react';
 import './App.css';
+import handy from "./assets/bag_1.png";
+import stylish from "./assets/bag_2.png";
+import simple from "./assets/bag_3.png";
+import trendy from "./assets/bag_4.png";
+import brand from "./assets/brand.png";
+import story from "./assets/our_story.png";
+import Button from "./components/Buttons";
+import Products from "./components/Products";
+import Tiles from "./components/Tiles";
 
 function App() {
-  return (
-      <h1>Handbags & Purses</h1>
-  );
+    return (
+        <>
+            <h1>Handbags & Purses</h1>
+            <nav>
+                <Button buttonText="to the collection"/>
```

Comment:
Ik merk dat je de buttons de onClick functionaliteit niet meegegeven hebt. Komt dat doordat je erover heen gelezen hebt/je code niet kon runnen of is dat iets wat nog lastig is?